mock-kubernetes-client/client/client.go

```diff
@@ -745,8 +745,8 @@ func (client *MockClient) SetPodIntoFailed(context ctx.Context, object ctrlClien
 	return client.Update(context, pod)
 }
 
-// RemovePodIP sets the IP address of the Pod to an empty string
-func (client *MockClient) RemovePodIP(pod *corev1.Pod) error {
+// RemoveProcessGroupIP sets the IP address of the Pod to an empty string
+func (client *MockClient) RemoveProcessGroupIP(pod *corev1.Pod) error {
```

Comment:
This should still be `RemovePodIP` since we are acting on the Pod level here. The `process group` is only a logical construct to group multiple resources like Pod, PVC, Service to map to the same underlying FoundationDB process.

Reply:
Reverted the change